lib/styles/components/pane.scss

```diff
@@ -1,4 +1,4 @@
-.bp3-drawer.nui-pane--wrapper {
+.bp3-drawer.nh-pane--wrapper {
```

Comment:
@yedhink _a name should be `nui` instead of `nh`

Reply:
Good catch. :+1: